resources/robots/magni/magni.json

```diff
@@ -0,0 +1,19 @@
+{
+    "magni":{
+        "icons":{
+            "magni": "/robots/magni/magni.png"
+        }, 
+        "places":[ "supplies",
```

Comment:
Correct me if I'm remembering wrong, but did we agree that the items in `places` will be a tuple of the place's name along with the dispenser name if any?

Reply:
Thanks for the feedback!
Yes, but I just wanted to show you this to see if there was any possibility of separating concepts. But it actually seems to be pretty correlated concepts. I feel like I'm missing something so let me check this with you if this affirmation is correct:
 just certain types (fleets) of robots can use the dispensers and that's why we should add the places and the dispensers inside the fleet and no as a general place-dispenser file.

Following the previous logic and as we discussed before, I'll follow this structure 
```
[{
place:NameOfPlace, 
dispensers:[dispenser1, dispenser2] 
},{}]
```

Reply:
Agreed.